plugins/actix-web/src/lib.rs

```diff
@@ -229,10 +231,16 @@ where
     /// recorded by the wrapper and serves them in the given path
     /// as a JSON.
     pub fn with_json_spec_at(mut self, path: &str) -> Self {
-        self.inner = self.inner.service(
-            actix_web::web::resource(path)
-                .route(actix_web::web::get().to(SpecHandler(self.spec.clone()))),
-        );
+        self.inner = self
+            .inner
+            .service(
+                actix_web::web::resource(path)
+                    .route(actix_web::web::get().to(SpecHandler(self.spec.clone()))),
+            )
+            .service(
```

Comment:
This call can now be feature-gated with `#[cfg(feature = "swagger-ui")]`

Reply:
Hi
unfortunetelly im stuck in here. I can configure Cargo.toml with that feature.
But this works only when "swagger-ui" feature is enabled. When its disabled I'm getting compile error, cause service expects 1 parameter:
```rust
self.inner = self
            .inner
            .service(
                actix_web::web::resource(path)
                    .route(actix_web::web::get().to(SpecHandler(self.spec.clone()))),
            )
            .service(
                #[cfg(feature = "swagger-ui")]
                actix_web::web::resource(path.to_owned() + "/swagger-ui.html")
                    .route(actix_web::web::get().to(SwaggerUIHandler(path.to_string()))),
            );
        self
```

if I move feature-gate up (before "service" ), then I'm getting syntax error - its not supported by the language.

Of course tried to make separate call to self.inner, but, as you can guess, compiler does not allow it. ownership problem. I'm a bit tiered for today. Need to repeat this lection about ownership in rust.
But if you can help here, I appreciate that

Reply:
You can try this?

```rust
self.inner = self
            .inner
            .service(
                actix_web::web::resource(path)
                    .route(actix_web::web::get().to(SpecHandler(self.spec.clone()))),
            );
#[cfg(feature = "swagger-ui")]
self.inner = self
            .inner
            .service(
                actix_web::web::resource(path.to_owned() + "/swagger-ui.html")
                    .route(actix_web::web::get().to(SwaggerUIHandler(path.to_string()))),
            );
self
```

Reply:
Hi, sorry for such a long delay. Was too far from all of this.
But yes. I tried this before. And tried this one more time right now. It raises compile error:
```error[E0658]: attributes on expressions are experimental```

Reply:
No worries. In that case, it could be moved to two private helper functions below. One is no-op, for when the feature is disabled, another which contains the actual logic for the feature.

Reply:
yes. right. I will do this